app/assets/javascripts/discourse/app/templates/components/groups-form-interaction-fields.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if canAdminGroup}}
   <div class="control-group">
     <label class="control-label">{{i18n "admin.groups.manage.interaction.visibility"}}</label>
-    <label for="visiblity">{{i18n "admin.groups.manage.interaction.visibility_levels.title"}}</label>
+    <label for="visibility">{{i18n "admin.groups.manage.interaction.visibility_levels.title"}}</label>
```

Comment:
Should this be `"alias"`?

Reply:
Wearing a different hat (one that I haven't used in a while)...

https://html.spec.whatwg.org/multipage/forms.html#the-label-element

`<label for=...>` is an id-ref, and there is no obvious sign of an `id=` attribute in the subsequent content:
https://github.com/discourse/discourse/blob/838fa12f14afadd3e01434ba6c2cb54d2afda0e8/app/assets/javascripts/discourse/app/templates/components/groups-form-interaction-fields.hbs#L6-L14

Yes, `for="visibility"` while the obvious typo fix is not the correct fix for this, assuming that the goal is to reference the highlighted `combo-box` (which seems likely).

I suspect that a correct fix to this is beyond the scope of this PR (it's huge already). I can drop this correction and split it off into an issue for someone else to fix (possibly me, but, likely not me in the near future -- I'm trying to make a release myself). (Or I can leave this spelling correction in and still split off an issue for someone to investigate all `<label for=...>` tags.)

I could eventually try to run a screen reader and test things, but, I think it's best for me to suggest that discourse do a11y testing w/ one instead.

Reply:
Moving to https://github.com/jsoref/discourse/commits/label-for-alias https://github.com/jsoref/discourse/commit/8e0fb512a81cd97e2b5d6b4ec215496c3667992b for now.

Reply:
#12813